language/stdlib/modules/vector.mvir

```diff
@@ -22,8 +22,11 @@ module Vector {
   // Pop an element from the end of vector, abort if the vector is empty.
   native public pop_back<Element>(v: &mut Self.T<Element>): Element;
 
-  // Return true if the vector has no elements
+  // Return true if the vector has no elements.
   public is_empty<Element>(v: &Self.T<Element>): bool {
     return Self.length<Element>(move(v)) == 0;
   }
+
+  // Destroy the vector, abort if not empty.
+  native public destroy_empty<Element>(v: Self.T<Element>);
```

Comment:
Nit: move above `is_empty`? I think this file is slightly easier to read if we write `native` methods first and then methods with bodies?.